openedx/core/djangoapps/self_paced/migrations/0001_initial.py

```diff
@@ -19,7 +19,6 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('change_date', models.DateTimeField(auto_now_add=True, verbose_name='Change date')),
                 ('enabled', models.BooleanField(default=False, verbose_name='Enabled')),
-                ('enable_course_home_improvements', models.BooleanField(default=False, verbose_name='Enable course home page improvements.')),
```

Comment:
Please see [Everything About Database Migrations](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Howtodropacolumn) document to check the process of the column removal. To avoid situation like [this](https://openedx.slack.com/archives/CSAC4QHC7/p1591672127105600) we should remove all existing code that uses this field and create a follow-up to introduce a new migration that will drop this column in the [Lilac](https://docs.google.com/spreadsheets/d/11DheEtMDGrbA9hsUvZ2SEd4Cc8CaC4mAfoV8SVaLBGI/edit#gid=417264012) release.

Reply:
I think this was a mistake: I had intended to not touch migrations as I realized that doing so would cause problems. I will revert this change.

Reply:
@Agrendalath, @taniwha, the "multiple releases" mentioned in that "Everything About Database Migrations" document is an edx.org release and not an Open edX release - I had discussed this with Ned and Troy Sankey on Open edX Slack [here](https://openedx.slack.com/archives/CSAC4QHC7/p1591811716118200?thread_ts=1591672127.105600&cid=CSAC4QHC7) and they confirmed this there.

Reply:
Thanks for the clarification, @lgp171188!